js/views/stock.js

```diff
@@ -8,6 +8,7 @@ StockView.prototype = {
   lookupSymbol: function() {
     var symbol = $("#symbol-input").val();
     this.model = new Stock(symbol);
+    this.model.getStock();
     this.render();
```

Comment:
The reason you have to click twice for this to work is that `getStock()` happens asynchronously. `this.render()` is happening before the stock has been "gotten". So the second time you click, it's updating with the information from the previous AJAX request.